locales_dev/en/translation.json

```diff
@@ -143,6 +143,7 @@
   "Epic Name": "Epic Name",
   "Epic Options": "Epic Options",
   "Epic Scratch Org": "Epic Scratch Org",
+  "Epic Status": "Epic Status",
```

Comment:
I believe these new strings need to be copied over to `locales/en/translation.json` to be included in the next localization pass.